src/applications/hca/tests/unit/utils/helpers/file-attachments.unit.spec.js

```diff
@@ -1,41 +1,34 @@
 import { expect } from 'chai';
 import { createPayload, parseResponse } from '../../../../utils/helpers';
 
-describe('hca `createPayload` method', () => {
-  let mockFile;
+describe('file-attachments', () => {
+  const mockFile = new File(['test'], 'test.txt', { type: 'text/plain' });
 
-  beforeEach(() => {
-    mockFile = new File(['test'], 'test.txt', { type: 'text/plain' });
-  });
+  describe('hca `createPayload` method', () => {
```

Comment:
This is being picky, but I also don't like nested `describes`. Can these nested ones be renamed into `context`?

Reply:
Okay, I'll make that change shortly!